packaging/opensuse/syslog-ng.spec

```diff
@@ -0,0 +1,1083 @@
+#
+# spec file for package syslog-ng
+#
+# Copyright (c) 2015 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+Name:           syslog-ng
+Version:        4.8.3
+Release:        2%{?dist}
+Summary:        The new-generation syslog-daemon
+License:        GPL-2.0
+Group:          System/Daemons
+Url:            https://syslog-ng.org/
+Source0:        https://github.com/balabit/%{name}/releases/download/%{name}-%{version}/%{name}-%{version}.tar.gz
+Source1:        %{name}.sysconfig
+Source2:        %{name}.conf.default
+Source3:        %{name}.service
+Source4:        %{name}-service-prepare
+
+%global		py_ver	 %(rpm -qf %{_bindir}/python3 --qf "%%{version}" | awk -F. '{print $1"."$2}')
+
+%bcond_without	redis
+%bcond_without	http
+%bcond_without	smtp
+%bcond_without	python
+%bcond_without	mqtt
+%bcond_without	bpf
+
+# missing dependencies on SLES 15
+%if 0%{?sle_version} == 150000 && !0%{?is_opensuse}
```

Comment:
Please clarify: The conditional would suggest these dependencies are indeed present on Leap 15.

Reply:
we do not package suse yet, this file is just a preparation for later use, once we need this, we can fix it